_format/1.1/index.md

```diff
@@ -1781,6 +1822,183 @@ parameter from this specification, it **MUST** return `400 Bad Request`.
+
+Likewise, clients and servers applying profiles to a JSON API document **MUST** 
+include a [top-level][top level] [`links` object][links] with a `profile` key, 
+and that `profile` key **MUST** include a [link] to the URI of each profile 
+that has been applied.
+
+When an older JSON API server that doesn't support the `profile` media type 
+parameter receives a document with one or more profiles, it will respond with a
+`415 Unsupported Media Type` error.
+
+After attempting to rule out other possible causes of this error, a client that
+receives a `415 Unsupported Media Type` **SHOULD** remove the profile extensions
+it has applied to the document and retry its request without the `profile` media
+type parameter. If this resolves the error, the client **SHOULD NOT** attempt to
+use profile extensions in subsequent interactions with the same API.
+
+> The most likely other causes of a 415 error are that the server doesn't
+support JSON API at all or that the client has failed to provide a required 
+profile.
+
+### <a href="#profile-query-parameter" id="profile-query-parameter" class="headerlink"></a> `profile` Query Parameter
+
+A client **MAY** use the `profile` query parameter to _require_ the server to
+apply one or more profiles when processing the request. The value of the `profile` 
+query parameter **MUST** equal a URI-encoded whitespace-separated list of profile URIs.
+
+If a server receives a request requiring the application of a profile or
+combination of profiles that it can not apply, it **MUST** respond with a `400
+Bad Request` status code.
+
+> Note: When a client lists a profile in the `Accept` header, it's asking the server
+> to compute its response as normal, but then send the response document with some
+> extra information, as described in the requested profile. By contrarst, when a client 
+> lists a profile in the `profile` *query parameter*, it's asking the server to *process
+> the incoming request* according to the rules of the profile.
+
+### <a href="#profile-descriptors" id="profile-descriptors" class="headerlink"></a> Profile Descriptors
+
+As described above, the top-level [`jsonapi`](document-jsonapi-object) object
+**MAY** contain a `profiles` member that contains a map of profile descriptor
+objects, keyed by profile URI.
+
+Each profile descriptor object **MAY** contain an `aliases` object. The key of
+each member in `aliases` **MUST** be a [keyword](profile-keywords) explicitly
+declared by the profile, and the value **MUST** be an alias that applies to this
+particular representation. This aliasing mechanism allows profiles to be applied
+in a way that is both consistent with the rest of the representation and does
+not conflict with other profiles.
+
+For instance, the following document includes a single profile descriptor for a
+profile that, let's say, assigns meaning to a `version` keyword that can be
+included in any resource's `meta` object. This descriptor provides an alias for
```

Comment:
Once we have a mechanism to explicitly declare a keyword, that also has as a side benefit that that declaration could also state whether the keyword applies to `meta`, a query param, or something else still.